curriculumcourse/src/main/java/curriculumcourse/curriculumcourse/CourseSchedule.java

```diff
@@ -47,6 +46,10 @@
 
     private java.util.List<Course> courseList;
 
+    @javax.annotation.Generated(value = {"org.optaplanner.workbench.screens.domaineditor.client.widgets.planner.PlannerDataObjectEditor"})
+    @org.kie.api.definition.type.Label(value = "Generated Planner score field")
+    private org.optaplanner.core.api.score.buildin.hardsoft.HardSoftScore score;
```

Comment:
@mcimbora Does the workbench always set a @PlanningSolution annotation on the score or not?
For bendable score it will have (no other way to do it).
For non bendable scores, the auto discovery takes care of it, so adding it is optional.